frontend/src/components/projects/projectSearchResults.js

```diff
@@ -16,10 +17,18 @@ export const ProjectSearchResults = ({
   retryFn,
   management,
   showBottomButtons,
+  isExploreProjectsPage = false,
 }) => {
   const listViewIsActive = useSelector((state) => state.preferences['projectListView']);
+  const isExploreProjectsTableView = useSelector(
+    (state) => state.preferences['isExploreProjectsTableView'],
+  );
+
   const cardWidthClass = 'w-100';
-  const isShowListView = management && listViewIsActive;
+  let isShowListView = management && listViewIsActive;
```

Comment:
this may be written as - 
```
const isShowListView =
  (management && listViewIsActive) || (isExploreProjectsPage && isExploreProjectsTableView);
```